src/vtlengine/Operators/Numeric.py

```diff
@@ -390,3 +381,37 @@ def py_op(cls, x: float, param: Optional[float]) -> Any:
             return truncated_value
 
         return int(truncated_value)
+
+
+class PseudoRandom(_random.Random):
```

Comment:
Maybe use random.Random instead of internal module _random

Reply:
_random contains the random class prototypes, so for class inheritance it is better to use _random as it generates fewer errors